frappe/core/doctype/recorder_request/recorder_request_list.js

```diff
@@ -0,0 +1,47 @@
+frappe.listview_settings["Recorder Request"] = {
+	hide_name_column: true,
+
+	onload(listview) {
+		listview.page.sidebar.remove();
+		if (!has_common(frappe.user_roles, ["Administrator", "System Manager"])) return;
+
+		frappe
+			.xcall("frappe.core.doctype.recorder_request.recorder_request.get_status")
+			.then((status) => {
+				if (status) {
+					listview.page.set_indicator(__("Active"), "green");
+				} else {
+					listview.page.set_indicator(__("Inactive"), "red");
+				}
+			});
+
+		listview.page.add_button(__("Start"), () => {
```

Comment:
Start/stop buttons should be primary and only one (relevant one) should be shown at a time.